### PR TITLE
[WIP] Intly 3133

### PR DESCRIPTION
**Pull request by damienomurchu**
intly-3133 - allow integreatly-operator to autodiscover the console url and routing subdomain

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,8 +47,7 @@ setup/service_account:
 	@oc replace --force -f deploy/role.yaml -n $(NAMESPACE)
 	@oc replace --force -f deploy/service_account.yaml -n $(NAMESPACE)
 	@oc replace --force -f deploy/role_binding.yaml -n $(NAMESPACE)
-	@oc replace --force -f deploy/clusterrole.yaml
-	@cat deploy/cluster_role_binding.yaml | sed "s/namespace: integreatly/namespace: $(NAMESPACE)/g" | oc replace --force -f -
+	@cat deploy/role_binding.yaml | sed "s/namespace: integreatly/namespace: $(NAMESPACE)/g" | oc replace --force -f -
```
Comment (damienomurchu) [inline Makefile]: Updating Makefile as everything consolidated in `role.yaml` & `role_binding.yaml` now

File: pkg/apis/integreatly/v1alpha1/installation_types.go [hunk 2]
```diff
@@ -68,8 +68,8 @@ var (
 	OperatorVersionAMQOnline             = "1.2.2"
 	OperatorVersionMonitoring            = "0.0.28"
 	OperatorVersionSolutionExplorer      = "0.0.32"
-	OperatorVersionRHSSO                 = "1.9.4"
-	OperatorVersionRHSSOUser             = "1.9.4"
+	OperatorVersionRHSSO                 = "1.9.5"
+	OperatorVersionRHSSOUser             = "1.9.5"
```
Comment (damienomurchu) [inline pkg/apis/integreatly/v1alpha1/installation_types.go]: Until Keycloak version is bumped in master

Comment (philbrookes) [reply pkg/apis/integreatly/v1alpha1/installation_types.go]: this should be done now, if you rebase on master

**Comment by philbrookes** [comment]
Is this still valid, or has @jjaferson opened a second PR for this?

**Comment by damienomurchu** [comment]
Closing this in favour of a PR that will be pushed by @jjaferson soon

**Pull request closed without merging**